web/src/components/Header/Header.tsx

```diff
@@ -1,64 +1,33 @@
-import {QuestionCircleOutlined} from '@ant-design/icons';
 import {useTour} from '@reactour/tour';
+import {Typography} from 'antd';
 import Logo from 'assets/Logo.svg';
-import {DOCUMENTATION_URL, GITHUB_URL} from 'constants/Common.constants';
-import {FC} from 'react';
+import {FC, useState} from 'react';
 import {Link, useLocation} from 'react-router-dom';
-import HomeAnalyticsService from 'services/Analytics/HomeAnalytics.service';
+import HomeAnalyticsService from '../../services/Analytics/HomeAnalytics.service';
 import * as S from './Header.styled';
+import './index.css';
+import {HeaderMenu} from './HeaderMenu';
 
 const {onGuidedTourClick} = HomeAnalyticsService;
 
 const Header: FC = () => {
   const {pathname} = useLocation();
   const {setIsOpen} = useTour();
-
-  const handleGuidedTourCLick = () => {
-    setIsOpen(true);
-    onGuidedTourClick();
-  };
-
+  const [visible, setVisible] = useState(false);
```

Comment:
If this state is going to be only used within the. HeaderMenu for bot reading and setting it, let's move it down to that component.

Reply:
ok